getting-started/kubernetes/flannel/migration-from-flannel.md

```diff
@@ -1,64 +1,102 @@
 ---
-title: Migrate a cluster from flannel to Calico
-description: Upgrade from legacy flannel VXLAN to Calico VXLAN. 
+title: Migrate a cluster from flannel networking to Calico networking
+description: Take advantage of the more flexible Calico IP address management by upgrading from flannel VXLAN to Calico VXLAN netowrking. 
```

Comment:
I am thinking the value proposition could be divided into two categories. 
1. Migrating from Flannel to Calico. User will take full advantage of Calico IPAM and advanced network policy features.
2. Migrating from Canal to Calico. Calico IPAM is the major reason for the migration. 
My understanding is most users belongs to the first category.